test/test_pdo.py

```diff
@@ -55,10 +65,23 @@ def test_bit_mapping(self):
         self.assertEqual(node.tpdo[0x2002].raw, 0xf)
         self.assertEqual(node.pdo[0x1600][0x2002].raw, 0xf)
 
-    def test_save_pdo(self):
-        node = canopen.Node(1, EDS_PATH)
-        node.tpdo.save()
-        node.rpdo.save()
+    def test_pdo_save(self):
+        self.node.tpdo.save()
+        self.node.rpdo.save()
+
+    @unittest.skipIf(canmatrix is None, "Requires canmatrix dependency")
+    def test_pdo_export(self):
+        import tempfile
+
+        for pdo in "tpdo", "rpdo":
+            with tempfile.NamedTemporaryFile(suffix=".csv") as tmp:
+                fn = tmp.name
+                with self.subTest(pdo=pdo, filename=fn):
```

Comment:
Perhaps a little more easy on the eyes when debugging a failed test:

```suggestion
                with self.subTest(filename=fn, pdo=pdo):
```